%% BioMed_Central_Tex_Template_v1.06
%%                                      %
%  bmc_article.tex            ver: 1.06 %
%                                       %

%%IMPORTANT: do not delete the first line of this template
%%It must be present to enable the BMC Submission system to
%%recognise this template!!

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%                                     %%
%%  LaTeX template for BioMed Central  %%
%%     journal article submissions     %%
%%                                     %%
%%          <8 June 2012>              %%
%%                                     %%
%%                                     %%
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%


%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%                                                                 %%
%% For instructions on how to fill out this Tex template           %%
%% document please refer to Readme.html and the instructions for   %%
%% authors page on the biomed central website                      %%
%% http://www.biomedcentral.com/info/authors/                      %%
%%                                                                 %%
%% Please do not use \input{...} to include other tex files.       %%
%% Submit your LaTeX manuscript as one .tex document.              %%
%%                                                                 %%
%% All additional figures and files should be attached             %%
%% separately and not embedded in the \TeX\ document itself.       %%
%%                                                                 %%
%% BioMed Central currently use the MikTex distribution of         %%
%% TeX for Windows) of TeX and LaTeX.  This is available from      %%
%% http://www.miktex.org                                           %%
%%                                                                 %%
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

%%% additional documentclass options:
%  [doublespacing]
%  [linenumbers]   - put the line numbers on margins

%%% loading packages, author definitions

%\documentclass[twocolumn]{bmcart}% uncomment this for twocolumn layout and comment line below
\documentclass{bmcart}

%%% Load packages
%\usepackage{amsthm,amsmath}
%\RequirePackage{natbib}
%\RequirePackage{hyperref}
\usepackage[utf8]{inputenc} %unicode support
%\usepackage[applemac]{inputenc} %applemac support if unicode package fails
%\usepackage[latin1]{inputenc} %UNIX support if unicode package fails


%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%                                             %%
%%  If you wish to display your graphics for   %%
%%  your own use using includegraphic or       %%
%%  includegraphics, then comment out the      %%
%%  following two lines of code.               %%
%%  NB: These line *must* be included when     %%
%%  submitting to BMC.                         %%
%%  All figure files must be submitted as      %%
%%  separate graphics through the BMC          %%
%%  submission process, not included in the    %%
%%  submitted article.                         %%
%%                                             %%
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%


\def\includegraphic{}
\def\includegraphics{}



%%% Put your definitions there:
\startlocaldefs
\endlocaldefs


%%% Begin ...
\begin{document}

%%% Start of article front matter
\begin{frontmatter}

\begin{fmbox}
\dochead{Research}

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%                                          %%
%% Enter the title of your article here     %%
%%                                          %%
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

\title{A sample article title}

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%                                          %%
%% Enter the authors here                   %%
%%                                          %%
%% Specify information, if available,       %%
%% in the form:                             %%
%%   <key>={<id1>,<id2>}                    %%
%%   <key>=                                 %%
%% Comment or delete the keys which are     %%
%% not used. Repeat \author command as much %%
%% as required.                             %%
%%                                          %%
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

\author[
   addressref={aff1},                   % id's of addresses, e.g. {aff1,aff2}
   corref={aff1},                       % id of corresponding address, if any
   noteref={n1},                        % id's of article notes, if any
   email={sstevens2@wisc.edu}   % email address
]{\inits{SLR}\fnm{Sarah LR} \snm{Stevens}}
\author[
   addressref={aff1},
   email={jjhamilton2@wisc.edu}
]{\inits{JH}\fnm{Joshua J} \snm{Hamilton}}
%ADD TRINA ET AL!!!!!


%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%                                          %%
%% Enter the authors' addresses here        %%
%%                                          %%
%% Repeat \address commands as much as      %%
%% required.                                %%
%%                                          %%
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

\address[id=aff1]{%                           % unique id
  \orgname{Department of Bacteriology, University of Wisconsin-Madison}, % university, etc
  %\street{Waterloo Road},                     %
  %\postcode{}                                % post or zip code
  \city{Madison},                              % city
  \cny{USA}                                    % country
}

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%                                          %%
%% Enter short notes here                   %%
%%                                          %%
%% Short notes will be after addresses      %%
%% on first page.                           %%
%%                                          %%
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

\begin{artnotes}
%\note{Sample of title note}     % note to the article
\note[id=n1]{Equal contributor} % note, connected to author
\end{artnotes}

\end{fmbox}% comment this for two column layout

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%                                          %%
%% The Abstract begins here                 %%
%%                                          %%
%% Please refer to the Instructions for     %%
%% authors on http://www.biomedcentral.com  %%
%% and include the section headings         %%
%% accordingly for your article type.       %%
%%                                          %%
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

\begin{abstractbox}
  Authors should provide a concise, non-redundant and meaningful abstract that describes the nature of the article. It should summarize the rationale, the objectives and the findings of the report and provide key details (e.g.,  relevant INSDC identifiers, culture collection identifiers, other project metadata that is accessible in standardized form).
\begin{abstract} % abstract
\parttitle{First part title} %if any
Text for this section.

\parttitle{Second part title} %if any
Text for this section.
\end{abstract}

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%                                          %%
%% The keywords begin here                  %%
%%                                          %%
%% Put each keyword in separate \kwd{}.     %%
%%                                          %%
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

Authors should include five to seven descriptive keywords. These may include the article type, the name(s) of the organism(s) sequenced, the next higher taxonomic rank, the sampling site and other significant details about the nature of the study. (Format Keywords)

\begin{keyword}
\kwd{sample}
\kwd{article}
\kwd{author}
\end{keyword}

% MSC classifications codes, if any
%\begin{keyword}[class=AMS]
%\kwd[Primary ]{}
%\kwd{}
%\kwd[; secondary ]{}
%\end{keyword}

\end{abstractbox}
%
%\end{fmbox}% uncomment this for twcolumn layout

\end{frontmatter}

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%                                          %%
%% The Main Body begins here                %%
%%                                          %%
%% Please refer to the instructions for     %%
%% authors on:                              %%
%% http://www.biomedcentral.com/info/authors%%
%% and include the section headings         %%
%% accordingly for your article type.       %%
%%                                          %%
%% See the Results and Discussion section   %%
%% for details on how to create sub-sections%%
%%                                          %%
%% use \cite{...} to cite references        %%
%%  \cite{koon} and                         %%
%%  \cite{oreg,khar,zvai,xjon,schn,pond}    %%
%%  \nocite{smith,marg,hunn,advi,koha,mouse}%%
%%                                          %%
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

%%%%%%%%%%%%%%%%%%%%%%%%% start of article main body
% <put your article body there>

%%%%%%%%%%%%%%%%
%% Introduction %%
%%
\section*{Introduction}
%Authors are expected to provide readers with brief, high-level description of the source organism and the rationale for its selection for sequencing. It should be written in such a manner as to capture a readerâ€™s attention. It should also indicate whether or not the organism is part of a larger genomic survey project.
Freshwater members of the family Comamonadaceae were found to fall within two monophyletic genera Rhodoferax and Limnohabitans, with members of Limnohabitans being unique to freshwater.\cite{Newton2011b}  These two genera are sometimes known as the BAL47 cluster or betI lineage.\cite{Newton2011b}  This group contains four previously published genomes all coming from freshwater aquatic ecosystems, two belonging to Rhodoferax and two belonging to Limnohabitans.  Limnohabitans is not only unique to freshwater but also ubiquous as it has been found in over 90\% of 161 european freshwater habitats and often makes up at least 29-30\% of bacteria by FISH.\cite{Jezbera2013,Simek2010a,Kasalicky2013}  Rhodoferax has also been detected by 16S rRNA gene sequencing in 22 of 27 lakes assayed, across three continents.\cite{Zwart2002,Lindstrom2005a}
The genomes included in this publication were part of two larger sequencing projects.  Single-cell genomes were originally chosen to represent a variety of common and abundant freshwater lineages.  Metagenome-assembled genomes (MAGs) were sequenced, assembled, and binned by the DOE's Joint Genome Institute using samples from an ongoing long-term ecological research project (LTER).


%%%%%%%%%%%%%%%%
%% Abbreviations (optional) %%
%%
\section*{Abbreviations}
Authors should include any non-standard abbreviations that are used throughout the article. Do not include well-known abbreviations (e.g., NCBI, EMBL, DNA, RNA) and do not use non-standard abbreviations for organism names. Species and subspecies names must be fully spelled out on first use as binomials (genus name and species epithet) or trinomials (genus name, species epithet subsp. subspecific epithet). Following first usage, the genus name may be abbreviated by using the first letter of the genus name, followed by a period and the epithets.


%%%%%%%%%%%%%%%%
%% Organism Information %%
%%
\section*{Organism Information}


\subsection*{Classification and features}
This should include succinct but detailed description of major phenotypic features (macro- and micromorphology, physiological characteristics), natural habitat, distribution, current classification and phylogenetic placement of the strain/specimen selected for sequencing. Authors should provide readers with additional background as to how the organism was isolated from nature (if a microbe), its association with other community members, any special properties that are noteworthy (e.g., pathogenic, symbiotic, industrial use, taxonomic type strain, model organism, etc.). In a separate subsection, of this section, authors should provide chemotaxonomic information (e.g. whole cell fatty acid composition, respiratory quinones, cell-wall composition, other unique or diagnostic cellular components).

This section should also include two figures: a phylogenetic tree indicating current placement and a photomicrograph or electron photomicrograph of the source organism. This section must also include a reference to Table 1, which provides a standardized summary of key features of the source organism. The layout of Table 1 is fixed and authors must not vary the appearance of information in the table. Rather, they must supply this information so that readers may view the descriptive information in a consistent fashion.

\subsubsection*{Chemotaxonomic data}
Optional
See the above description (under Classification and features)

\subsubsection*{Symbiotaxonomy}
Optional
See the above description (under Classification and features)


%%%%%%%%%%%%%%%%
%% Genome sequencing information %%
%%
\section*{Genome sequencing information}

\subsection*{Genome project history}
%This section of the manuscript should provide a detailed summary of the sequencing, assembly and annotation methodology.  The section should include an introductory paragraph that provides the readers with specific information about the sequencing project, when the project began and was completed, whether the sequence is complete or remains as a draft genome, and the quality of the draft, which public databases contain the project data and other relevant information. These data should be summarized in Table 2.
AAA027-C02 and the metagenomes used in the assembly and binning of the composite genomes were selected for seuqencing due to the importance of freshwater bacteria in global carbon cycling and were sequenced as part of the Community Science Program at the U.S. Department of Energy, Joint Genome Institute (JGI).  Sequencing and annotation were performed by the JGI and metadata and high quality draft genome sequence were deposited in the Genomes OnLine Database [REFERENCE?] and in IMG/GEBA.  A summary of the project information is shown in Table 3(?Do we have this table? Does it have this information it?).

\subsection*{Genome sequencing and assembly}
%Provide a succinct and detailed description of the methods used to sequence and assemble the genome(s). Identify the sequencing center where the work was performed, the sequencing technology(ies) used, library construction, number of reads and read length. Cite any relevant references regarding methods used. Also, provide a succinct and detailed description of the assembly, including the software used for preliminary assembly, finishing and error checking and correction of mis-assemblies.  Provide a brief description of the size of the final assembly, the number of contigs, and coverage.
\subsubsection*{Single-cell genomes}
AAA027-C02 was sorted and underwent multipule displacement amplification (MDA) at the Single Cell Genomics Center [REFERENCE].  Sequencing was performed on both Illumina HiSeq 2000 and Illumina HiSeq 2500 platforms.  ### XXX type of reads were generated.  Assebmly was preformed with QQQ software and average genome coverage for AAA027-C02 was YYY.  The total assembled size was Based on an analysis of 139 genes univesral, single copy genes, AAA027-C02 was estimated to be about ZZ\% complete. All aspects of the sequencing can be found in Table 2.

\subsubsection*{Genomes from metagenomes}
XX and YY filtered water samples were collected from the hypolimnion and epilimnion of Trout Bog Lake, respectively.  The libraries prepared from these filteres were sequenced using the Illumina YYYY platform.


\subsection*{Genome annotation}
Provide a brief and succinct description of the methods used to identify and annotate genes, and any software used in the annotation pipeline.

%%%%%%%%%%%%%%%%
%% Genome Properties  %%
%%
\section*{Genome Properties}
Provide a summary description of the size of the genome(s) (in base pairs), the number of chromosomes and plasmids. Include the number of predicted genes (RNA genes, protein coding genes, pseudogenes) by number and percent of total. This section should be linked to a chromosome map, map(s) of any plasmids and two or three tables providing a more detailed summary of the genome properties.

%%%%%%%%%%%%%%%%
%% Insights from the genome sequence (optional) %%
%%
\section*{Insights from the genome sequence}
In many cases, authors may wish to provide a brief, yet more detailed description of major findings arising from the genome sequence. This can be a comparison of major differences found between the genome sequence that is the subject of the study and others (e.g., major differences is specific metabolic pathways, significant differences in gene content, etc.). This section is intended to permit the authors to make preliminary observations rather than to serve as detailed comparative study. In a short genome report, this section should be limited to two to three paragraphs. Authors wanting to provide greater detail and to incorporate additional genomes into their study, or to incorporate additional tables and figures are invited to submit their articles as extended genome reports.

\subsection*{Extended insights}
Authors are encouraged to provide more detailed descriptions about insights gained from the genome sequence. This may include comparisons of the genome to that of closely relate species, detailed discussions about specific metabolic pathways that are noteworthy or unique, or other features that may be of interest to the readers. Authors may include additional tables and figures in this section (see below).

%%%%%%%%%%%%%%%%
%% Conclusions %%
%%
\section*{Conclusions}

%%%%%%%%%%%%%%%%
%% Taxonomic and nomenclatural proposals (optional)  %%
%%
\section*{Taxonomic and nomenclatural proposals}

Authors are free to make taxonomic proposals and revisions of existing taxa, providing that the proposals are made in accordance with the rules of the relevant code of nomenclature. Taxonomic proposals must include the following sections appearing after the Conclusions section: A formal description for each taxon - Each taxonomic proposal must have its own subsection heading, and must appear in the proper order. Proposals for new genera must precede proposals for new species or subspecies. New species must precede new subspecies. A proposed name and etymology â€“ For each new taxon proposed authors must propose a new name, in accordance to the appropriate rules of nomenclature. The proposed name should be followed by the etymology of the name, in grammatically correct Latin. Authors are responsible for ensuring that proposed names meet these requirements.
A protologue â€“ for each new taxon and name that is proposed, authors must provide a description (also referred to as a diagnosis in botany) that provides readers with a summarized statement of differential features that can be used to distinguish the proposed taxon from other, closely related taxa. The protologue should include information about the morphology, physiology, habitat and genetics, along with any marker genes or features that can be used for identification purposes. The protologue must conclude with a statement that positively establishes the type strain (prokaryotes) or specimen (botany and zoology). If a new species or subspecies of bacteria or archaea is proposed, authors must provide the accession numbers from at least two internationally recognized culture collections (in different countries) from which viable samples of the type strain are available without restriction. Proposals that fail to provide this information cannot be considered for valid publication. If one or more new genera are proposed, the genus name(s) and description(s) must precede those of newly proposed member species. Genus descriptions must indicate the type species of the genus and differential/diagnostic features. In many cases, these may be the same as that for member species. Proposals for novel higher taxa (family and above) should appear after the species or subspecies proposals. Emendations of existing taxa should be made in separate sections, indicating the changes in membership and phenotypic and genotypic characteristics on which the taxa were originally formed. Assertions of synonymy should be presented in a separate section, with a full description of which taxa are being combined and an assertion of which name has priority. Taxonomic proposals of eukaryotic and virus taxa will follow the same general outline described above, but the identification and deposition of type material differs.

%%%%%%%%%%%%%%%%
%% Authors' contributions  %%
%% Might not be the right heading level
\section*{Authors' contributions}
In order to give appropriate credit to each author of a paper, the individual contributions of authors to the manuscript should be specified in this section.

According to ICMJE guidelines(), An 'author' is generally considered to be someone who has made substantive intellectual contributions to a published study. To qualify as an author one should 1) have made substantial contributions to conception and design, or acquisition of data, or analysis and interpretation of data; 2) have been involved in drafting the manuscript or revising it critically for important intellectual content; 3) have given final approval of the version to be published; and 4) agree to be accountable for all aspects of the work in ensuring that questions related to the accuracy or integrity of any part of the work are appropriately investigated and resolved. Each author should have participated sufficiently in the work to take public responsibility for appropriate portions of the content. Acquisition of funding, collection of data, or general supervision of the research group, alone, does not justify authorship.

We suggest the following kind of format (please use initials to refer to each author's contribution): AB carried out the molecular genetic studies, participated in the sequence alignment and drafted the manuscript. JY carried out the immunoassays. MT participated in the sequence alignment. ES participated in the design of the study and performed the statistical analysis. FG conceived of the study, and participated in its design and coordination and helped to draft the manuscript. All authors read and approved the final manuscript.

%%%%%%%%%%%%%%%%
%% Acknowledgements   %%
%%
\section*{Acknowledgements}
Authors are encouraged to include an acknowledgement section recognizing the contributions to their work made by non-authors (skilled technicians, contributors of materials or those providing specialized advice or commentary on various aspects of their work. Authors should also acknowledge the source of funding for their work provided by various agencies, foundations or benefactors as well as the relevant contract or agreement numbers.


\end{document}
